mods/ra/maps/soviet-05/soviet05.lua

```diff
@@ -8,8 +8,8 @@
 ]]
 
 CheckForBase = function()
-	baseBuildings = Map.ActorsInBox(Map.TopLeft, CFBPoint.CenterPosition, function(actor)
```

Comment:
The previous code (and `CheckForCYard` below) are checking specific regions of the map, whereas `CheckForSPen` is using `Map.ActorsInWorld`. This suggests that the limited search regions are intentional.

I suggest changing back to `ActorsInBox`, renaming `CFBPoint` to `BaseRectBR`, and adding an `BaseRectTL` waypoint to replace `Map.TopLeft` to ensure that there are no gameplay changes.